examples/simple_key_value_store.rs

```diff
@@ -184,8 +184,10 @@ impl Sendable for TestData {
 
     fn serialised_contents(&self) -> Vec<u8> {
         let mut e = cbor::Encoder::from_memory();
-        e.encode(&[&self]).unwrap();
-        e.into_bytes()
+        match e.encode(&[&self]) {
+            Ok(()) => e.into_bytes(),
+            Err(_) => Vec::<u8>::new()
+        }
```

Comment:
If an error happens when encoding, the client will send an empty vector of bytes. I don't think this is the correct behavior (is it?). In an example I would suggest to either `assert`, which would be the simplest solution. I think the ideal solution would be for this function to return Result<Vec<u8>, some-cbor-error-type>, that way, the code that uses this function can handle the situation and just not send anything. The ideal solution might be an overkill here though as encoding will hardly ever fail in this example.